main/src/main/java/mockit/internal/faking/FakedClassModifier.java

```diff
@@ -118,12 +117,13 @@ public MethodVisitor visitMethod(int access, @Nonnull String name, @Nonnull Stri
             return cw.visitMethod(access, name, desc, signature, exceptions);
         }
 
-        if (isPrivate(access)) {
-            String kindOfMember = isConstructor ? "constructor " : "method ";
-            String privateMemberDesc = fakedClass.getSimpleName() + '#' + name + desc;
-            throw new IllegalArgumentException(
-                    "Unsupported fake for private " + kindOfMember + privateMemberDesc + " found");
-        }
+        // Supported fake for private method
```

Comment:
@yukkes Can you add a simple further note here that we are keeping commented out code from original changes?  Normally I'd say delete it but I think in this case I agree commenting it out is fine but want to make sure its obvious why we are keeping commented out code, otherwise someone will delete it it in the future.